### PR TITLE
Moved pokemessages into a seperate file and added more.

### PR DESCRIPTION
**Pull request by grilledham**

### CODE REVIEW

File: locale/resources/regulars.lua [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    zacman0510 = "",
+    TheOrangeAngle = "",
+    williambellwisdo = "",
+    Extraxyz = "",
+    markupolionCZ = "",
+    BJHunter = "",
+    ZTX = "",
+    bigglesthepirate = "",
+    ["-slug-"] = "",
+    ["g-h-o-s-t"] = "",
```
Comment (Valansch) [inline locale/resources/regulars.lua]: Did you try this? I'm pretty sure that this will not work. But I could be wrong.

Comment (grilledham) [reply locale/resources/regulars.lua]: image

Comment (Valansch) [reply locale/resources/regulars.lua]: TIL

**Comment by Valansch** [comment]
Thank you very much

**Pull request merged**